### ISSUE TITLE
[Feature Request] Allow multiple hints for different cloze spans that are part of the same merged cloze card

### ISSUE THREAD

**Issue by WildGenius** [bjsi/cloze-hints#3]
Currently, the same hint is applied for every span. See image.
![image](https://user-images.githubusercontent.com/8071762/191967938-92e12f9c-9817-4112-986c-266d3f739fea.png)

**Comment by hugomarins** [comment]
Indeed, this behavior is strange and makes the usefulness of hints for multiple clozes not good. I hope the team will implement this correction!

**Comment by WandersonRossi** [comment]
I would like this to be fixed too! It is a minor visual annoyance/clutter. And it prohibits having multiple clozes with different hints on the same rem.

**Comment by bourbon91c** [comment]
I agree that this would be great to be able to make hints that work better in our flash card reviews. Please implement if able
